### DISCUSSION TITLE
Keba P30 mit fiktivem Verbrauch

### DISCUSSION THREAD

**Discussion by hkleinbub** [camueller/SmartApplianceEnabler#433] [Q&A] [answered]
Hallo
meine Keba P30 zeigt im SHM einen sehr hohen, fiktiven Verbrauch für eine Ladesession an.
![grafik](https://user-images.githubusercontent.com/60285099/210850559-a56d8203-7914-4602-817c-0a66307d579e.png)
Es gab aber gar keine Ladesession. Das Auto war auch nicht angeschlossen.
![grafik](https://user-images.githubusercontent.com/60285099/210850413-f30a689b-5c2c-4eaa-b60d-7728cd998958.png)
Im SAE-Log erkenne ich nicht, was dazu führt. Kann das jemand erklären?

Viele Grüße
Holger

**Comment by camueller** [comment, marked as answer]
Suche mal im Log nach `PowerInfo` - das ist die Leistung, die dem SHM vom SAE gemeldet wurde - als mein Auto heute geladen hat, waren das um 10:52 Uhr ca. 4,2 kW:
```
2023-01-05 10:52:55,348 DEBUG [http-nio-8080-exec-6] d.a.s.s.w.SempController [SempController.java:362] F-00000001-000000000019-00: PowerInfo{averagePower=4218, minPower=null, maxPower=null, timestamp=0, averagingInterval=60}
```
Wenn dieser Wert nicht stimmt, musst Du prüfen, warum der Wert falsch ist. Vielleicht stimmen die Ziffern des Wertes, aber das Komma ist an der falschen Stelle: Dann hast Du den Umrechnungsfaktor falsch angegeben - siehe:
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Keba_DE.md

**Comment by hkleinbub** [comment]
Schon mal vielen Dank!
Habe 2 solcher Einträge gefunden. Wie gesagt, es gab aber gar keine Ladesession. Das Keba-Log zeigt auch keine an.
![grafik](https://user-images.githubusercontent.com/60285099/210859891-5b68b122-895d-4f08-8fe2-9a9830f6d90e.png)
Anbei die Konfiguration der Wallbox. Ich meine, dass ich mich an die Anleitung gehalten habe.
![grafik](https://user-images.githubusercontent.com/60285099/210860582-5ad58981-e0a3-414e-ab6c-2acb9aef22cc.png)

**Comment by camueller** [comment]
Du musst ja zusätzlich zur Wallbox auch noch den Zähler konfigurieren - siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Keba_DE.md

**Comment by hkleinbub** [comment]
Hallo Axel,

ist eigentlich alles aus der Vorkonfiguration übernommen und eingestellt. Habe mich strikt an deine Anleitung gehalten. Das Einzige, was fehlt, ist die Fahrzeugkonfiguration. Da bin ich gerade dran, dass Python-Skript für meine eGolf zu bauen, um den Ladestatus auszulesen.

Gruß

 

Von: Axel Müller ***@***.***> 
Gesendet: Freitag, 6. Januar 2023 06:54
An: camueller/SmartApplianceEnabler ***@***.***>
Cc: hkleinbub ***@***.***>; Author ***@***.***>
Betreff: Re: [camueller/SmartApplianceEnabler] Keba P30 mit fiktivem Verbrauch (Discussion #433)

 

Du musst ja zusätzlich zur Wallbox auch noch den Zähler konfigurieren - siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Keba_DE.md

**Comment by camueller** [comment]
Mit der dokumentierten Zählerkonfiguration wird ja der Zählerstand aus dem angegebenen Register ausgeladen. Da kannst ja im Log nachsehen, ob das so der Fall ist. Falls Du keinen Fehler in der Zählerkonfiguration hast, musst Du herausfinden, warum Deine Keba in diesem Register solche Werte liefert. Ist das ein Defekt oder hat Keba die Register geändert?
